Ref Guide - Home Overview (#96)
uyuni-project/uyuni-docs: modules/installation/pages/webui-setup.adoc

[[webui-setup]]
= Web Interface Setup

This topic introduces you to the {productname} {webui}.

Entering the {productname} URL in a browser leads you to the [guimenu]``Sign In`` screen.

> Before logging in, select [guimenu]``About``, to browse and search for available documentation topics.

Reviewer: ```suggestion
Before signing in, select [guimenu]``About``, to browse and search for available documentation topics.
```

You may reset your username and password from the menu:About[Lookup Login/Password] page.

> Refer to xref:reference:home/user-account-my-account.adoc[My Account] for more information.

Reviewer: ```suggestion
See xref:reference:home/user-account-my-account.adoc[My Account] for more information.
```

// Move the Navigation chapter from the Ref Guide into this document. LKB 2020-02-20
[NOTE]
====
New to the {productname} {webui}? Scan through xref:reference:navigation/navigation-menu.adoc[{WebUI} Navigation] to familiarize yourself with the layout and symbols used throughout this document.
====

> Login to the {productname} {webui} to view the menu:Main Menu[Home > Overview] page.

Reviewer: ```suggestion
Sign in to the {productname} {webui} to view the menu:Main Menu[Home > Overview] page.
```

Reviewer: Stopping with suggestions now.  Look unedited, at least partially.  systems instead of clients, "refer to" instead of "see".  menu:Main Menu.

Author: Yep, completely unedited ;)
